fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -257,6 +258,24 @@ std::pair<Version, uint32_t> decodeBKMutationLogKey(Key key) {
 	    bigEndian32(*(int32_t*)(key.begin() + backupLogPrefixBytes + sizeof(UID) + sizeof(uint8_t) + sizeof(int64_t))));
 }
 
+bool isSystemKey(KeyRef key) {
```

Comment:
After this change, the code will be duplicated at `ApplyMetadataMutation.h` and `BackupAgentBase`; can we instead move this method to a common file for both modules to use? Maybe `SystemData.h`.